srcpkgs/bochs/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'bochs'
 pkgname=bochs
-version=2.6.11
+version=2.7
 revision=1
 build_style=gnu-configure
-configure_args="--enable-debugger --enable-disasm --disable-readline"
+configure_args="--with-sdl2 --enable-all-optimizations --enable-x86-64 --enable-smp --enable-debugger --enable-debugger-gui --enable-avx --enable-evex --enable-x86-debugger --enable-usb --enable-usb-ohci --enable-usb-ehci --enable-usb-xhci --enable-e1000 --disable-readline"
 hostmakedepends="pkg-config"
-makedepends="libX11-devel libXrandr-devel gtk+-devel"
-short_desc="Highly portable open source IA-32 (x86) PC emulator"
+makedepends="libX11-devel libXrandr-devel SDL2-devel"
+short_desc="Highly portable open source x86(_64) PC emulator"
 maintainer="mid-kid <esteve.varela@gmail.com>"
```

Comment:
Please change the maintainer to `Orphaned <orphan@voidlinux.org>` since they haven't contributed since 2015.